### PR TITLE
Story/vgnwb 219

### PR DESCRIPTION
**Pull request by tjquinn1**
Moved lifecycle type ids to settings.py to allow them to be configured through export statements when necessary.

### CODE REVIEW

File: vogon/settings.py [hunk 1]
```diff
@@ -245,3 +245,15 @@
...
 
 SESSION_COOKIE_NAME = 'vogon'
+
+
+# Lifecycle types
+PERSONAL = os.environ.get('PERSONAL', '986a7cc9-c0c1-4720-b344-853f08c136ab')
+CORPORATE = os.environ.get('CORPORATE', '3fc436d0-26e7-472c-94de-0b712b66b3f3')
+GEOGRAPHIC = os.environ.get('GEOGRAPHIC', 'dfc95f97-f128-42ae-b54c-ee40333eae8c')
+
+LIFECYCLE_TYPES = {
```
Comment (jdamerow) [inline vogon/settings.py]: these are not lifecycle types but concept types.

File: concepts/lifecycle.py [hunk 2]
```diff
@@ -5,11 +5,9 @@
 
 from concepts.models import *
 
-TYPES = {
-    'viaf:personal': "986a7cc9-c0c1-4720-b344-853f08c136ab",    # E21 Person
-    'viaf:corporate': "3fc436d0-26e7-472c-94de-0b712b66b3f3",   # E40 Legal Body
-    'viaf:geographic': "dfc95f97-f128-42ae-b54c-ee40333eae8c"    # E53 Place
-}
+from django.conf import settings
```
Comment (jdamerow) [inline concepts/lifecycle.py]: this has already been imported in line one

**Review by jdamerow — CHANGES_REQUESTED** [review]

**Comment by jdamerow** [comment]
please resolve conflicts

**Comment by tjquinn1** [comment]
removed import and changed lifecycle to concept

**Pull request closed without merging**